docs/repositories/pull-requests.md

```diff
@@ -0,0 +1,46 @@
+---
+page_name: "pull request"
+file_name: "pull-requests"
+---
+
+# Pull requests page
```

Comment:
should we capitalize both words? The side panel shows Pull Requests, as does the dropdown to select the type of PR